### ISSUE TITLE
[Bug] Shapes: ArcSegment throwing on iOS, doing nothing on Android

### ISSUE THREAD

**Issue by davidbritch** [xamarin/Xamarin.Forms#11190]
### Description

I'd expect the following code to work:

```
        <Path Stroke="Black">
                <Path.Data>
                    <PathGeometry>
                        <PathGeometry.Figures>
                            <PathFigureCollection>
                                <PathFigure StartPoint="10,100">
                                    <PathFigure.Segments>
                                        <PathSegmentCollection>
                                            <ArcSegment Size="100,50"
                                                        RotationAngle="45"
                                                        IsLargeArc="True"
                                                        SweepDirection="CounterClockwise"
                                                        Point="200,100" />
                                        </PathSegmentCollection>
                                    </PathFigure.Segments>
                                </PathFigure>
                            </PathFigureCollection>
                        </PathGeometry.Figures>
                    </PathGeometry>
                </Path.Data>
            </Path>
```

On iOS this throws a `ArgumentOutOfRangeException`: Index was out of range. Must be non-negative and less than the size of the collection. Parameter name: index

On Android, nothing renders.

### Steps to Reproduce

1. Run the attached sample.
2. Browse to the PathGeometry page.
3. Scroll down to "PathGeometry with an ArcSegment".

### Expected Behavior

`ArcSegment` is drawn.

### Actual Behavior

iOS: `ArgumentOutOfRangeException`
Android: Nothing drawn.

### Basic Information

- Version with issue: 4.7.0.968
- IDE: VSMac 8.6.4
- Platform Target Frameworks: 
  - iOS:  13.5
  - Android: API 28

### Reproduction Link

[ShapesDemos.zip](https://github.com/xamarin/Xamarin.Forms/files/4826210/ShapesDemos.zip)

**Comment by eli191** [comment]
Hi, I have this issue, can you tell me if the fix is released please?

**Comment by davidbritch** [comment]
Hi @eli191 

It's fixed in 5.0 (which is currently a pre-release).

**Comment by eli191** [comment]
Super ! I will try it then !
